### PR TITLE
Logger

### PR DESCRIPTION
**Pull request by jethomasii**

### CODE REVIEW

File: kdlc/cli.py [hunk 1]
```diff
@@ -22,7 +21,29 @@
...
+        logger.add(
+            sys.stdout,
+            level="DEBUG",
+            format="<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | <cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> - <level>{message}</level>",
+        )
+    else:
+        logger.add(
+            sys.stderr,
+            level="ERROR",
+            format="<green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> | <cyan>{name}</cyan>:<cyan>{function}</cyan>:<cyan>{line}</cyan> - <level>{message}</level>",
```
Comment (kubejm) [inline kdlc/cli.py]: Maybe abstract this string out into a variable.  Seems like it's the same on lines 38 and 44.

File: kdlc/KDLLoader.py [hunk 2]
```diff
@@ -22,12 +24,14 @@ def exitNode_settings(
...
     ) -> None:
         node_number = ctx.node().node_id().getText()
-        # print(f"nodeNumber: {node_number}")
+        # logger.debug(f"nodeNumber: {node_number}")
 
         json_tokens = [i.getText() for i in ctx.json().children]
         json_string = "".join(json_tokens)
         node_settings = json.loads(json_string)
 
+        # logger.debug(node_settings)
```
Comment (kubejm) [inline kdlc/KDLLoader.py]: Probably good to uncomment the use of logger in here and rip the print statements.

File: kdlc/cli.py (hunk 1 above)
Comment (kubejm) [reply kdlc/cli.py]: Actually... looking at this again, I think we might just need the if condition.  It seems there isn't a use of logger.error, so I'd probably remove it for now.

File: kdlc/KDLLoader.py (hunk 2 above)
Comment (jethomasii) [reply kdlc/KDLLoader.py]: Do we need these statements or will it be redundant based on what we have in Commands.py ?

File: kdlc/cli.py (hunk 1 above)
Comment (jethomasii) [reply kdlc/cli.py]: Resolved: https://github.com/k-descriptor-language/kdl/commit/fefb4be17976d72bed0d87a9f9ba1401878c084f

File: kdlc/KDLLoader.py (hunk 2 above)
Comment (jethomasii) [reply kdlc/KDLLoader.py]: Resolved by https://github.com/k-descriptor-language/kdl/commit/e06043de395ebd398dd90572e0dae8ffbc433267

**Pull request merged**